### ISSUE TITLE
[Bug] Several projects in `./examples` do not run due to missing client

### ISSUE THREAD

**Issue by bmcilw1** [wundergraph/wundergraph#303]
### Bug description

The example projects do not run. Wundergraph does not generate all the files it used to, or they have been renamed.

<img width="247" alt="Screen Shot 2022-10-28 at 7 53 52 PM" src="https://user-images.githubusercontent.com/1582095/198754466-1194444e-154b-4571-8e15-e20b6aeb099d.png">

### How to reproduce

1. Run `nvm use --lts` and `node --version` to ensure node 16.x
2. Clear node cache to rule out any cached strange behavior `npm cache clean --force`
1. Download repo `git clone https://github.com/wundergraph/wundergraph`
3. Navigate to example `cd wundergraph/examples/nextjs-postgres-user-filters` (or any example with a seed script)
4. Run `npm install && npm start` and wait until console finishes tasks
5. Open another terminal in the same directory and run `npm run check`
6. See errors in the console

### Expected behavior

Can the master branch be kept stable so folks can use the examples at any time? If that is the intention, it seems like adding tests that run the CLI and check that the expected files are generated would be a great addition to tests.

### WunderGraph information

Same as in this repo. Here are some snippets in question. `wundergraph.client` is not generated.

error
```ts
npm run check

> wundergraph-nextjs-postgres-prisma@0.1.0 check
> tsc --noEmit

seed/seed.ts:1:24 - error TS2307: Cannot find module '../.wundergraph/generated/wundergraph.client' or its corresponding type declarations.

1 import { Client } from '../.wundergraph/generated/wundergraph.client';
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in seed/seed.ts:1
```

~~Here are also errors in React. This error occurs ONLY the first time running `npm run start` and is not repeated after the client is generated.~~ Edit - this wasn't reproduced for me after deleting my `node_modules`, but the error above was repeated.

### Environment & setup

- OS:  Mac 12.6 Intel chip
- Go version: 
- Database:  Postgres
- Node.js version:  16

### WunderCtl Version

```
npx wunderctl version
Version: 0.114.2
Commit: ee87a7d44e3ca7c5cb9c0c54d96aba85b7a1346b
Date: 2022-10-22T23:57:39Z
BuiltBy: ci
```

**Comment by bmcilw1** [comment]
As I've looked deeper into the repo, the CLI, underlying commands, etc, I see that the example repos are not the only place that could use a bit more test coverage lovin' 🙁 😬

<img width="699" alt="Screen Shot 2022-10-28 at 8 24 07 PM" src="https://user-images.githubusercontent.com/1582095/198755559-67cda192-9cec-434f-ab82-9f8b4c9f79b3.png">

**Comment by fiam** [comment]
@bmcilw1 I managed to reproduce something similar to what you're experiencing, but after deleting `node_modules` and `package-lock.json` I cannot reproduce it anymore. Please, try deleting those and rebuilding.

Regarding the tests for the Go packages, some of those are very simple ones which consist of generated code (like e.g. `templates` or `wgpb`) or are examples (like `testapps/mtls/{client,server}`. I agree that having more tests will always be better, but it's not like we have none. You can't just look at the number of packages with/without tests and conclude anything from there, because some packages are bigger/more complex than others.

**Comment by bmcilw1** [comment]
After deleting `node_modules` and `package-lock.json`, clearing my npm cache, etc. I've tried again. The `wundergraph.client` file is still not being generated for me.

**Comment by fiam** [comment]
Fixed by #321

**Comment by fiam** [comment]
@bmcilw1 Thanks for the heads up! Some of the generated files were renamed but I had them generated with an older version of the SDK, that's why I couldn't reproduce initially. We took the chance to go through all the examples and make sure they were up to date.
